api/batch/manifest.go

```diff
@@ -219,16 +219,14 @@ func (m *manifestManager) DeleteDriverAuthorization(ctx context.Context, namespa
 	return nil
 }
 
-func (m *manifestManager) CreateSecret(ctx context.Context, predictionJobName string, namespace string, data string) (string, error) {
+func (m *manifestManager) CreateSecret(ctx context.Context, predictionJobName string, namespace string, secretMap map[string]string) (string, error) {
```

Comment:
nit: same as before to prevent confusion between method names xD

Reply:
Thanks for suggesting this change! 😅 I've also renamed `CreateSecret -> CreateK8sSecret` and `DeleteSecret -> DeleteK8sSecret`.